packages/markers/src/browser/marker-manager.ts

```diff
@@ -197,4 +225,10 @@ export abstract class MarkerManager<D extends object> {
         }
     }
 
+    cleanMarkersByOwner(owner: string): void {
+        this.uri2MarkerCollection.forEach((collection, uri) => {
+            collection.removeMarkers(owner);
+            this.fireOnDidChangeMarkers(new URI(uri));
```

Comment:
How do you that event should be fired? Does calling `removeMarkers` removes somethign always? Please add tests.